libs/filamat/src/shaders/CodeGenerator.cpp

```diff
@@ -40,7 +40,7 @@ io::sstream& CodeGenerator::generateSeparator(io::sstream& out) {
 }
 
 utils::io::sstream& CodeGenerator::generateProlog(utils::io::sstream& out, ShaderStage stage,
-        MaterialInfo const& material) const {
+        MaterialInfo const& material, filament::Variant v) const {
```

Comment:
Should we pass the variant here, or should it be a field in `MaterialInfo`?

Reply:
I thought about that, but `MaterialInfo` contains information about the material as a whole, while use of `gl_ClipDistance` will depend on the specific variant.